heon/automate_full_test.sh

```diff
@@ -0,0 +1,27 @@
+# Quickstart script for running a full test
+# The qdisc test (qdisc_test.sh) should be running before this script is 
+# executed so that different qdiscs can be tested
+
+run () {
+	echo "[$1]"
+	read -p "Once ready, press any key to continue... " -n1 -s
+	echo ""
+}
+
+run "Full Test Automation"
+
+python3 generate_flow_seq.py
+
+run "Random & Desired Flows Generated"
+
+(trap 'kill 0' SIGINT; python3 multi_server.py & python3 multi_client.py)
```

Comment:
This `trap` command kills all processes started by bash if you press Ctrl-C. Is that the intention?
Why is this useful? A short note (comment) may be helpful.

Reply:
I intended this because the server can only be manually ended. I thought that would be the reasonable way to implement it as the server may not know when the client stops sending packets. I will add a comment to explain that.

Reply:
(Server automatically terminates even without ctrl+c)
(This could be a problem if the server terminates in the middle of the interaction) 
(Make sure the trap command does exactly what I intend it to)